src/containers/MasonryLayout.jsx

```diff
@@ -18,6 +26,9 @@ export class MasonryLayout extends Component {
     ));
   }
 
+  /**
+   * React's lifecycle method componentDidMount
+   */
   componentDidMount() {
     this.props.items.forEach((item, index) => {
       const div = document.createElement('div');
```

Comment:
Should we use jsx syntax for creating elements rather than using javascript

Reply:
No. JSX just provides syntactic sugar for the React.createElement(component, props, ...children) function. In this case, we need an element to render to.

Reply:
Why render it here and not in the render function?

Reply:
We can't add items to columns before the columns are rendered.